config/exclude_files.json

```diff
@@ -1 +1,96 @@
-{}
\ No newline at end of file
+{
+  "red-hat-data-services/data-science-pipelines-operator":{
+    ".github/*": "",
+    "datasciencecluster/*":"",
+    "docs/*":"",
+    "hack/*":"",
+    "scripts/*":"",
+    "tests/*":""
+  },
+  "red-hat-data-services/data-science-pipelines":{
```

Comment:
@HumairAK we could also add `requirements.txt` here, looks like a file we don't touch and I see some 10 HL and 4 critical CVEs for it on our snyk board.